narwhal/node/tests/reconfigure.rs

```diff
@@ -159,7 +159,7 @@ async fn run_client(
         }
     }
 }
-
+#[ignore]
```

Comment:
can this be fixed, or what's the status of it? maybe add a comment explanation if you intend to keep it ignored on the criteria for eventually either deleting or unignoring?